pom.xml

```diff
@@ -147,6 +147,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-cps</artifactId>
+      <classifier>tests</classifier>
```

Comment:
This _replaces_ the test dep on `workflow-cps.jar`. I think you wanted to _add_ a dependency with this classifier.

Reply:
Added the classified dependency.